Swallowing exception in directConstructor


Original issue 532 created by davidsarah.hopwood on 2008-06-12T00:05:17.000Z:
In directConstructor in caja.js, we have
try {
[...]
} catch (ex) {
return (void 0);
}
We shouldn't be throwing the exception on the floor.
In general, caja.js etc. should be reviewed for similar cases.
